starcheck/pcad_att_check.py

```diff
@@ -139,7 +139,46 @@ def make_pcad_attitude_check_report(backstop_file, or_list_file=None, attitude_f
     # gives the history of updates as a dict with a `value` and `date` key.
     sc = hopper.run_cmds(backstop_file, or_list, ofls_characteristics_file,
                          initial_state=initial_state, starcheck=True)
-    # Iterate through checks by obsid to print status
+
+    # Make maneuver structure
+    mm = []
+    for m in sc.maneuvers:
+        q1 = Quat(q=Quaternion.normalize(
+            [m['initial']['q1'], m['initial']['q2'],
+             m['initial']['q3'], m['initial']['q4']]))
+        q2 = Quat(q=Quaternion.normalize(
+            [m['final']['q1'], m['final']['q2'],
+             m['final']['q3'], m['final']['q4']]))
+
+        # Calculate maneuver angle using code borrowed from kadi
```

Comment:
I updated the angle calculation to just use that 4th component -- which seems to work and was already numerically stable here with the logic from kadi commands even if q2 == q1?